Dockerfile

```diff
@@ -1,14 +1,21 @@
-FROM mikefarah/yq
+#
+# Copyright (c) 2012-2019 Red Hat, Inc.
```

Comment:
It seems that license years are not accurate

Reply:
fixed, thanks